chimney/src/main/scala/io/scalaland/chimney/internal/macros/MappingMacros.scala

```diff
@@ -137,24 +144,36 @@ trait MappingMacros extends Model with TransformerConfiguration {
         }
 
       defaultValueFallback orElse optionNoneFallback orElse unitFallback
-    }.toMap
+    }
```

Comment:
`.to(ListMap)` doesn't work?

Reply:
failed on scala 2.12 and 2.11, even with `import scala.collection.compat._`